fictadvisor-web/src/app/error.tsx

```diff
@@ -0,0 +1,55 @@
+'use client';
+
+import React, { useEffect } from 'react';
+import { Box, Typography } from '@mui/material';
+import Link from 'next/link';
+
+import * as styles from '@/app/(main)/error-pages/MainErrorPage.styles';
+import PageLayout from '@/components/common/layout/page-layout';
+import Button from '@/components/common/ui/button-mui';
+import {
+  ButtonColor,
+  ButtonSize,
+  ButtonVariant,
+} from '@/components/common/ui/button-mui/types';
+
+export default function Error({
+  error,
+  reset,
+}: {
+  error: Error & { message?: string };
+  reset: () => void;
+}) {
+  useEffect(() => {
+    console.error(error);
+  }, [error]);
+
+  return (
+    <PageLayout hasFooter={false}>
```

Comment:
this component is completely the same as in /src/app/global-error.tsx, so move it to component and reuse in both pages